### PR TITLE
Toast and toad in a hole Fix

### PR DESCRIPTION
**Pull request by SamGondra**
#### Summary
Bugfixes "Fix/Update Toast and toad_in_a_hole"

#### Purpose of change
Makes the toad in a hole recipe use heat and a proper tool to actually cook. Also adds some Proficiencies and makes the buttered/jam/etc. Toasts use previously toasted bread instead of toasting the bread themselves.

#### Describe the solution
Made the toad-in-a-hole recipe a recipe on its own, since it needs a different cooking implement anyway than just just making toast, which you could conceivably do by using a skewer.
Also added Proficiencies and a bit of oil/butter for frying since I was at it, and gave the wheat free version all the wheat free breads.
Discovered that the different Jam/peanutbutter/butter versions all tried to copy from the recipe that was toasting the respective bread, but because of how copy from works in that it didn't copy over tools and qualities (I do not know how this works/if it ever worked like that). adjusted recipes to use the respective toasted bread instead, and tried to make sure everything has the right materials for fructose/grain/lactose intolerance.

#### Describe alternatives you've considered
Dropping all the extra recipes besides toasting bread and toad in the hole and just adding toast as possible (Bread)component for sandwiches, but that should probably increase fun of the products more than standard bread and my Brain is colse to rebelling anyway.
Also wondered if I should take crackers out of the toad in the hole recipe as possible bread, but brain said no.

#### Testing
Did make sure the recipes loaded in game and the recipes and products looked right.
("LIST" is a suprisingly important argument (Sorry again peeps in #dev-help))

### CODE REVIEW

**Comment by Inglonias** [comment]
Just a heads-up. The recipe described in game is not called "toad-in-a-hole" as far as I know. It's common name is "[eggs in a nest](https://en.wikipedia.org/wiki/Egg_in_the_basket)". [Toad in a hole is something different](https://en.wikipedia.org/wiki/Toad_in_the_hole).

**Comment by SamGondra** [comment]
> Just a heads-up. The recipe described in game is not called "toad-in-a-hole" as far as I know. It's common name is "[eggs in the basket](https://en.wikipedia.org/wiki/Egg_in_the_basket)". [Toad in a hole is something different](https://en.wikipedia.org/wiki/Toad_in_the_hole).
> 
> I figure as long as we're changing things a bit, we may as well fix the name...

I am aware of that and had asked in discord - the setting is in new England though and there is apparently an 'american' version with the same name that is how this recipe is described.
Although I guess I could add a blurb in the description noting the difference.

**Comment by Maleclypse** [comment]
> Just a heads-up. The recipe described in game is not called "toad-in-a-hole" as far as I know. It's common name is "[eggs in the basket](https://en.wikipedia.org/wiki/Egg_in_the_basket)". [Toad in a hole is something different](https://en.wikipedia.org/wiki/Toad_in_the_hole).
> 
> I figure as long as we're changing things a bit, we may as well fix the name...

It is called toad in a whole in parts of America and a quick google search would show you many of the recipes for it are the same as Egg in the basket.   https://www.huffpost.com/entry/toad-in-a-hole-egg-in-a-basket-worthy_n_6994866

If I google "toad in the hole" while the top three links are for classic english toad in the hole less than halfway down the first page it shows an egg in a hole in a piece of toast.   I'm wondering if that's altered for region, it's entirely possible.

**Comment by Maleclypse** [comment]
I left out my thoughts that the original Toad in the Hole would likely only be found in Classic English Cooking books rather than something that could be autolearned and might as well then be called "Classic English Toad in the Hole".

**Comment by SamGondra** [comment]
Added a remark in the description about the difference to the english version of the recipe, although it does not feel particular elegant.

**Comment by fairyarmadillo** [comment]
Why not name the food what it's generally called in America (eggs in a basket) since that's both where the game takes place and also the clearest solution? Then if someone wants to add proper toad in the hole later there's not a conflict.

File: data/json/items/comestibles/bread.json [hunk 1]
```diff
@@ -289,23 +297,29 @@
...
     "relative": { "calories": 80, "fun": 4, "flags": [ "FREEZERBURN" ], "vitamins": [ [ "iron", 4 ], [ "calcium", 3 ] ] }
   },
   {
     "type": "COMESTIBLE",
     "id": "toad_in_a_hole_wheat_free",
     "name": { "str_sp": "wheat-free toad in a hole" },
     "copy-from": "toast_wheat_free",
-    "description": "A cooked egg served inside a piece of wheat-free toast.",
-    "relative": { "calories": 80, "fun": 4, "flags": [ "FREEZERBURN" ], "vitamins": [ [ "iron", 4 ], [ "calcium", 3 ] ] }
+    "description": "A cooked egg served inside a piece of wheat-free toast, , not to be confused with the english recipe of the same name.",
```
Comment (Stone94) [inline data/json/items/comestibles/bread.json:310]: There's an extra comma there, and English should be capitalized.
```suggestion
    "description": "A cooked egg served inside a piece of wheat-free toast, not to be confused with the English recipe of the same name.",
```

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**